ambry-clustermap/src/main/java/com.github.ambry.clustermap/HardwareLayout.java

```diff
@@ -77,6 +81,10 @@ public long getVersion() {
     return datacenters;
   }
 
+  public Datacenter getDatacenterById(byte id) {
```

Comment:
add a javadoc for this method. Also, I might suggest calling this method `findDatacenter` to match the other method on line 198.

Reply:
will use findDatacenter to keep match the other. This is good, getDatacenter* is confused somehow.